src/System.Security.Cryptography.Pkcs/src/Configurations.props

```diff
@@ -2,9 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
       netstandard;
+      netcoreapp-Windows_NT;
+      netcoreapp;
```

Comment:
Based on your PR description I think it would be better to be netcoreapp-Unix instead of netcoreapp, assuming this is unix specific. If it is just IL and can run on any OS then it is fine to keep it as netcoreapp.